documentation/modules/ROOT/pages/transformations/outbox-event-router.adoc

```diff
@@ -208,6 +217,10 @@ ifdef::community[]
 The outbox event router SMT supports arbitrary payload formats. The `payload` column value in an outbox table is passed on transparently. An alternative to working with JSON is to use Avro.
 This can be beneficial for message format governance and for ensuring that outbox event schemas evolve in a backwards-compatible way.
 
+ifdef::product[]
```

Comment:
Strictly speaking, I don't think we need this, given the entire page is marked as TP at the top. But it doesn't do harm either.